interfaces/cuda/Control.cu

```diff
@@ -140,6 +140,12 @@ unsigned ConcreteAPI::getGlobMemAlignment() {
   return 128;
 }
 
+size_t ConcreteAPI::alignBytes(size_t requestedBytes) {
```

Comment:
Would getSizeAlignedBytes be more precise?

Reply:
I made this function a little bit more generic and gave it a more generic name:
https://github.com/SeisSol/Device/blob/e050438c703a10996c3429d2c0c2e8b583a8af1c/interfaces/common/Common.h#L21-L25